src/notifications/ios_plugin.md

```diff
@@ -0,0 +1,83 @@
+# Add the iOS Notification Plugin
```

Comment:
Perhaps we can link [breez-sdk-swift README](https://github.com/breez/breez-sdk-swift/blob/main/README.md) for other installation methods.